packages/sit-onyx/src/components/OnyxMobileNavButton/OnyxMobileNavButton.vue

```diff
@@ -16,35 +17,43 @@ const emit = defineEmits<{
 
 defineSlots<{
   /**
-   * Slot for the menu content when its open.
+   * Slot for the menu content when it's open.
    */
   default(): unknown;
 }>();
 </script>
 
 <template>
-  <div>
+  <div class="onyx-mobile-nav-button">
     <button
       type="button"
-      class="onyx-mobile-nav-button"
-      :class="{ 'onyx-mobile-nav-button--active': props.open }"
+      class="onyx-mobile-nav-button__trigger"
+      :class="{ 'onyx-mobile-nav-button__trigger--active': props.open }"
       :aria-label="props.label"
       @click="emit('update:open', !props.open)"
     >
       <OnyxIcon :icon="props.open ? x : props.icon" />
     </button>
 
-    <div v-if="props.open" class="onyx-mobile-nav-button__menu">
-      <div class="onyx-mobile-nav-button__content">
-        <slot></slot>
-      </div>
+    <template v-if="props.open">
+      <div class="onyx-mobile-nav-button__flyout">
+        <div
+          class="onyx-mobile-nav-button__menu"
+          :class="{ 'onyx-mobile-nav-button__menu--no-headline': !props.headline }"
```

Comment:
Why can't we set the padding on the menu/flyout itself instead of the headline?
Then this class would be obselete, right?